mr/src/test/java/org/apache/iceberg/mr/hive/HiveIcebergTestUtils.java

```diff
@@ -107,6 +113,13 @@
               PrimitiveObjectInspectorFactory.writableStringObjectInspector
           ));
 
+  public static final DateTimeFormatter timestampWithTZFormatter = new DateTimeFormatterBuilder()
```

Comment:
can we adjust the casing to align with the other static final fields in the class?

Reply:
Fixed, also fixed in `TestHiveIcebergStorageHandlerTimezone`